src/main/java/uk/nhs/hee/tis/revalidation/connection/service/MasterElasticSearchService.java

```diff
@@ -54,7 +54,7 @@ public MasterElasticSearchService(
     this.elasticsearchTemplate = elasticsearchTemplate;
   }
 
-  private final int PAGE_SIZE = 1000;
+  private static final int SCROLL_TIMEOUT_MS = 30000;
```

Comment:
Just wondering, @CaiWillis If we increase the size more than 10000, that might require increasing the heap memory size of the ES cluster

Reply:
This is not page size, this is a timeout for the amount of time we allow the scroll to run (see the method definition for searchScrollStart/searchScrollContinue - it was previously being misused)

I think that this will not increase the amount of data that is pulled in one go, just the duration that the while/for loop runs

Reply:
That's correct. I have found the signature of the method
searchScrollStart(long scrollTimeInMillis, Query query, Class<T> clazz, IndexCoordinates index) 

Happy to approve. @CaiWillis